### PR TITLE
Fix QF calculations

### PR DESCRIPTION
**Pull request by trizin**
Fix QF calculations

### CODE REVIEW

File: src/airtable/sync_airtable_all_proposal_votes_snapshot.js [hunk 1]
```diff
@@ -75,21 +66,23 @@ const getAllProposalVotes = async () => {
...
-  )
+  // await updateProposalRecords(proposalVoteSummary)
+  // Logger.log(
+  //   '[%s]\nUpdated [%s] rows to Airtable',
+  //   new Date().toString(),
+  //   proposalVoteSummary.length
+  // )
 }
 
 main()
```
Comment (idiom-bytes) [inline src/airtable/sync_airtable_all_proposal_votes_snapshot.js:88]: Ok, I agree. I don't believe this is being used anymore. I'll try to remove functionality as soon as it's deprecated. We can always do github code archeology.

Comment (idiom-bytes) [inline src/airtable/sync_airtable_all_proposal_votes_snapshot.js:88]: Ok, I agree. I don't believe this is being used anymore. I'll try to remove functionality as soon as it's deprecated. We can always do github code archeology.

File: src/gsheets/sync_gsheets_active_proposal_votes_snapshot.js [hunk 2]
```diff
@@ -439,7 +389,7 @@ const syncGSheetsActiveProposalVotes = async (
 
   // Output the round summary
   proposalSummary = await calculateProposalSummary(
-    activeProposals,
+    activeProposals, // ACTIVE PROPOSALS IS NOT DEFINED??
```
Comment (idiom-bytes) [inline src/gsheets/sync_gsheets_active_proposal_votes_snapshot.js:392]: Is this comment valid? Is this a TODO?

Comment (idiom-bytes) [inline src/gsheets/sync_gsheets_active_proposal_votes_snapshot.js:392]: Is this comment valid? Is this a TODO?

File: src/gsheets/sync_gsheets_active_proposal_votes_snapshot.js [hunk 3]
```diff
@@ -370,53 +367,6 @@ const dumpRoundSummaryToGSheets = async (
   await updateValues(oAuth, sheetName, 'J1:Q' + flatObj.length, flatObj)
 }
 
-// DRY
```
Comment (idiom-bytes) [inline src/gsheets/sync_gsheets_active_proposal_votes_snapshot.js:373]: Thank you.

Comment (idiom-bytes) [inline src/gsheets/sync_gsheets_active_proposal_votes_snapshot.js:373]: Thank you.

File: src/gsheets/sync_gsheets_active_proposal_votes_snapshot.js (hunk 2 above)
Comment (trizin) [reply src/gsheets/sync_gsheets_active_proposal_votes_snapshot.js:392]: Calling that function [here](https://github.com/oceanprotocol/DAOBot/blob/f284abf8d1c6a122e1238422225546de35d7421e/src/scripts/update_funding_round.js#L300) returns an error, I added it as a note.

File: src/snapshot/snapshot_utils.js [hunk 4]
```diff
@@ -494,34 +495,33 @@ const reduceProposalScores = (ballotType, voterScores) => {
...
-  let summed = 0; // Setup summed grant contributions
-
-  // Loop over each grant
-  for (const grant of Object.keys(granularVotes)) {
-    let sumAmount = 0;
-
-    // Sum the square root of each grant contribution
-    for (let j = 0; j < granularVotes[grant].length; j++) {
-      sumAmount += Math.sqrt(granularVotes[grant][j]);
+const calculateMatch = (reducedVoterScores) => {
```
Comment (idiom-bytes) [inline src/snapshot/snapshot_utils.js:498]: Thanks for the fix while reducing logic from tests.

Comment (idiom-bytes) [inline src/snapshot/snapshot_utils.js:498]: Thanks for the fix while reducing logic from tests.

**Review by idiom-bytes — COMMENTED** [review]
Great!

File: src/gsheets/sync_gsheets_active_proposal_votes_snapshot.js (hunk 2 above)
Comment (idiom-bytes) [reply src/gsheets/sync_gsheets_active_proposal_votes_snapshot.js:392]: Thank you!

**Pull request merged**